assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -5,12 +5,19 @@ import { getGalleryLayout, GALLERY_BLOCK_CLASSES } from './getGalleryLayout';
 import { Lightbox } from '../../components/Lightbox/Lightbox';
 import { useLightbox } from '../../components/Lightbox/useLightbox';
 
+const getTitle = (image) => {
+  const caption = image.caption || '';
+  const credits = image.credits && image.credits.length > 0 && ! caption.includes(image.credits)
```

Comment:
I'm pretty sure that you can remove the `image.credits.length > 0` part, since an empty string will always return `false` 🤔 
https://stackoverflow.com/questions/8692982/in-javascript-is-an-empty-string-always-false-as-a-boolean